utils/trivialize-rules/trivialize-cookie-rules.js

```diff
@@ -65,7 +65,7 @@ let isTrivial = securecookie => {
     }
 
     for (let securecookie of securecookies) {
-      if (!securecookie.name === ".+") {
+      if (".+" !== securecookie.name) {
```

Comment:
```suggestion
      if (securecookie.name !== ".+") {
```

Reply:
I agree this looks better, but I am not sure why they don't seem to work the same on my machine though...

Reply:
@cschanaj Add code like:
```js
if ((securecookie.name !== ".+") !== (".+" !== securecookie.name)) {
  console.log(securecookie.name)
}
```
and run your script. Most likely reason is name being empty or undefined.

Reply:
thanks for your suggestion. It turns out that I might made a mistake when comparing the output of the script.